src/Notepads/Controls/TextEditor/TextEditorCore.cs

```diff
@@ -935,6 +935,35 @@ private static int LastIndexOfWholeWord(string target, int startIndex, string va
             return -1;
         }
 
+        public string GetSearchString()
+        {
+            var searchString = Document.Selection.Text.Trim();
+
+            if (searchString.Contains(RichEditBoxDefaultLineEnding.ToString())) return string.Empty;
+
+            if (string.IsNullOrEmpty(searchString) && Document.Selection.StartPosition < _content.Length)
+            {
+                var startIndex = Document.Selection.StartPosition;
+                var endIndex = Document.Selection.StartPosition;
```

Comment:
Document.Selection.EndPosition

Reply:
This case is only triggered either when no text is selected(In that case it won't matter) or when only spaces, tabs, line ending characters selected in that case it would produce unnecessary results. I would suggest to keep it same.